.github/scripts/end2end/configs/mongodb_options.yaml

```diff
@@ -448,7 +448,10 @@ metrics:
 
   ## String with extra arguments to the metrics exporter
   ## ref: https://github.com/percona/mongodb_exporter/blob/master/mongodb_exporter.go
-  extraArgs: ""
+  ##
+  ## We use "--compatible-mode" flag to export old metric: "mongodb_mongod_replset_oplog_head_timestamp"
+  ## for mongoDB and backbeat dashboards.
+  extraArgs: "--collect-all --compatible-mode"
```

Comment:
looking at the code for mongodb-exporter, the collect-all flag will activate the scanning of every collection and each index metric : i.e. it will list the collections, then iterate over them to get collection-specific metrics and expose this to prometheus.

this may be an issue:
* This may increase load (scan all collections), in case we have many collections/buckets
* This may generate high-cardinality metrics, if the collection is used as a label

Can you confirm this?

Reply:
note: there are multiple more granular --collect-xxxx flags we may use instead

Reply:
After discussion, I went for the "more granular" options.